mods/ts/weapons/healweapons.yaml

```diff
@@ -1,25 +1,25 @@
-Heal:
+^HealWeapon:
 	ReloadDelay: 80
 	Range: 2c849
 	Report: healer1.aud
-	ValidTargets: Infantry
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
```

Comment:
At least on my end the idea was to have a projectile that doesn't have/need any speed, angle or graphics calculations.

But even if `InstantHit` passed the target actor, without additional changes the `TargetActor` warhead would only be compatible with that projectile, which doesn't sound great to me, either.